From: Miroslav Benes <mbenes@suse.cz>
Date: Mon, 5 Jan 2015 13:29:41 +0100
Subject: kgr: fix typo in error message
Patch-mainline: submitted for review
References: fate#313296

Fix typo in error message during creation of kgraft directory in sysfs

Signed-off-by: Miroslav Benes <mbenes@suse.cz>
Reviewed-by: Petr Mladek <pmladek@suse.cz>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/kgraft_files.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/kgraft_files.c b/kernel/kgraft_files.c
index f6b4eae6b764..6164bbe87126 100644
--- a/kernel/kgraft_files.c
+++ b/kernel/kgraft_files.c
@@ -182,7 +182,7 @@ int kgr_add_files(void)
 
 	kgr_sysfs_dir = kobject_create_and_add("kgraft", kernel_kobj);
 	if (!kgr_sysfs_dir) {
-		pr_err("kgr: cannot create kfraft directory in sysfs!\n");
+		pr_err("kgr: cannot create kgraft directory in sysfs!\n");
 		return -EIO;
 	}
 
-- 
2.2.1

